src/components/DpSlidebar/DpSlidebar.mdx

```diff
@@ -9,3 +15,9 @@ Put content inside the default slot.
 ## Roadmap
 
 - Remove data attributes in favor of refs
+
+```html
+<dp-slidebar>
+    Test
+</dp-slidebar>
+```
```

Comment:
I think we should wait before merge until we figured out how to handle the slots propperly...